internal/utility/vulns/vulnerability_test.go

```diff
@@ -704,3 +704,32 @@ func TestOSV_IsAffected_OnlyVersions(t *testing.T) {
 	// an empty version should always be treated as affected
 	expectIsAffected(t, vuln, "", true)
 }
+
+func TestOSV_EcosystemsWithSuffix(t *testing.T) {
+	t.Parallel()
+
+	vuln := buildOSVWithAffected(
+		models.Affected{
+			Package: models.Package{Ecosystem: "Debian:12", Name: "my-package"},
+			Ranges: []models.Range{
+				buildSemverAffectsRange(
+					models.Event{Introduced: "0"},
```

Comment:
what do two introduced events mean?

Reply:
they were copied from another test - at the end of the day I don't think it's a big deal having them as it represents a more "real world" OSV, but I've removed everything all the same 🤷